plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/Exception.hs

```diff
@@ -148,13 +171,7 @@ instance (HasPrettyDefaults config ~ 'True, Pretty fun) =>
 
 instance
         ( HasPrettyDefaults config ~ 'True
-        , PrettyBy config internal, Pretty user
-        ) => PrettyBy config (EvaluationError user internal) where
-    prettyBy config (InternalEvaluationError err) = fold
-        [ "error:", hardline
-        , prettyBy config err
-        ]
-    prettyBy _      (UserEvaluationError err) = fold
-        [ "User error:", hardline
-        , pretty err
-        ]
+        , Pretty operational, PrettyBy config structural
+        ) => PrettyBy config (EvaluationError operational structural) where
+    prettyBy _      (OperationalEvaluationError operational) = pretty operational
+    prettyBy config (StructuralEvaluationError structural)   = prettyBy config structural
```

Comment:
And indeed here you show that it does not matter if the error was operational or structural :P

Reply:
It matters when `extractEvaluationResult` or `unsafeToEvaluationResult` are called and they do get called. I'm not sure if reporting whether the error was operational or structural would add meaningful context or just confuse the reader who may not be familiar with the distinction. I do want _us_ to be familiar with the distinction.